pkg/v2/release.go

```diff
@@ -50,6 +58,8 @@ func GetReleaseVersions(retOpts RetrieveOptions) ([]*rls.Release, error) {
 		return nil, fmt.Errorf("%s has no deployed releases\n", retOpts.ReleaseName)
 	}
 
+	sort.Sort(ByReleaseVersion(releases))
```

Comment:
For the sake of simplicity, is the `reverse` method at the end of this file still needed, now that release are sorted?

Reply:
Good point. Removed.